models/projection.rb

```diff
@@ -9,6 +9,13 @@ def calculate
     end
   end
 
+  def self.my_projections(ids, week)
+    player_ids = ids.split('-').map(&:to_i)
+    week = week || Projection.maximum(:week)
```

Comment:
Good or statement here to allow for finding projections for the current week easily.